com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/DownsampleDepth.shader

```diff
@@ -60,9 +61,15 @@ Shader "Hidden/HDRP/DownsampleDepth"
         float4 _DstOffset;
 #endif
 
+        float4 _ScaleBias; // x: uv offset x, uv offset y, uv x scale, uv y scale,
+
         void Frag(Varyings input, out float outputDepth : SV_Depth)
         {
             UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
+#if GREATER_THAN_HALF_DOWNSAMPLE
+            float4 depths = GATHER_RED_TEXTURE2D_X(_CameraDepthTexture, s_trilinear_clamp_sampler, input.texcoord * _ScaleBias.xy + _ScaleBias.zw);
```

Comment:
No need for trilinear, this doesn't have mips, linear is enough

Reply:
Done